### ISSUE TITLE
Try many sample programs

### ISSUE THREAD

**Issue by kollokollo** [dmsc/fastbasic#42]
Since fastbasic is similar to TURBO BASIC XL, and TURBO BASIC XL is similar to GFA-BASIC, and GFA-BASIC is similar to X11-Basic, you might want to try out some of the sample programs from the collection: https://codeberg.org/kollo/X11-Basic_examples

I think at least some of them can be also ised with FASTBASIC to check its performance. Also there is a small tool which can remove the line numbers from ANSI BASIC programs and thus help adapt alien dialects. 

Just to let you know. Have fun!

**Comment by eahumada** [comment]
It not possible to add support for line number to use turbo basic listings out-the-box ?

**Comment by kollokollo** [comment]
As far as I remember, TURBO BASIC XL did not require line numbers. (maybe they where optional). In GFA-Basic line numbers finally where not used anymore.

**Comment by eahumada** [comment]
Of course, but many basic examples that you want to try online has line numbers, then requires some re-work to take them and try with fastbasic ex. http://atariprojects.org/wp-content/uploads/2021/03/Page-Flipping.txt

**Comment by eahumada** [comment]
I created a web page where you can try the included samples online:
https://eahumada.github.io/AtariOnline/fastbasic.html?disk_filename=fastbasic.atr
When the emulator loads complete, you can try:
PMTEST.COM

**Comment by kollokollo** [comment]
I have read the article "Converting to Fastbasic" in the wiki. The process described is OK, however the process can be automated. I did this once, therefor I have written a little tool (in BASIC) which can convert ANSI BASIC (with line numbers and multiple commands in one line) to X11-Basic (or GFA-Basic) similar to fastbasic syntax. Maybe you want to have a look at it to write something similar. Bas2x11basic really has helped me converting some of the BASIC code of the ancient days... Here is it:
https://codeberg.org/kollo/X11Basic/src/branch/master/examples/compiler/bas2x11basic.bas
